src/LineItems.tsx

```diff
@@ -7,6 +7,7 @@ import { CertificationStatuses, RefundStatuses } from "./constants";
 import { DownloadLink } from "./DownloadLink";
 import ExIcon from "./ExIcon";
 import { ILink, LineItem } from "./LineItem";
+import * as TH from "./TH";
```

Comment:
For future devs, 
```suggestion
import * as TableHeader from "./TableHeader";
```
might be more clear